
 cache Pulling 
 api1 Pulling 
 api2 Pulling 
 consumer1 Pulling 
 9824c27679d3 Already exists 
 cache Error toomanyrequests: You have reached your unauthenticated pull rate limit. https://www.docker.com/increase-rate-limit
 api2  Interrupted
 api1  Interrupted
Error response from daemon: toomanyrequests: You have reached your unauthenticated pull rate limit. https://www.docker.com/increase-rate-limit
